src/static/riot/competitions/tile/competition_tile.tag

```diff
@@ -36,7 +36,7 @@
 
         self.pretty_description = function(description){
             if (description) {
-                description = render_markdown(description)
+                description = render_markdown(description).replaceAll("<p>", "").replaceAll("</p>", "")
```

Comment:
should we even need this? shouldn't `<p>` markdown get rendered as normal HTML fine? do we need to add `<p>` as a whitelisted tag?

Reply:
It doesn't seem like it does. This is the current [codabench](www.codabench.org) page
![image](https://user-images.githubusercontent.com/16579931/98313545-eef55180-1f88-11eb-9c40-e04aa027f433.png)

Reply:
is the "rendering of markdown" even working then? does that description get rendered properly elsewhere?

Reply:
It looks like the `render_markdown` was actually unnecessary in the first place. Updated the code.